tests/hazmat/primitives/utils.py

```diff
@@ -491,19 +488,29 @@ def kbkdf_counter_mode_test(backend, params):
     supported_counter_locations = {
         "before_fixed": CounterLocation.BeforeFixed,
         "after_fixed": CounterLocation.AfterFixed,
+        "middle_fixed": CounterLocation.MiddleFixed,
     }
 
     ctr_loc = supported_counter_locations[params.pop("ctrlocation")]
+    brk_loc = None
+
+    if ctr_loc == CounterLocation.MiddleFixed:
+        assert "fixedinputdata" not in params
+        params["fixedinputdata"] = params.pop("databeforectrdata")
+        params["fixedinputdata"] += params.pop("dataafterctrdata")
```

Comment:
```suggestion
        params["fixedinputdata"] = params.pop("databeforectrdata") + params.pop("dataafterctrdata")
```

Reply:
FWIW, I tried this approach, but `black` reformats it into:
```
-        params["fixedinputdata"] = params.pop("databeforectrdata")
-        params["fixedinputdata"] += params.pop("dataafterctrdata")
+        params["fixedinputdata"] = params.pop(
+            "databeforectrdata"
+        ) + params.pop("dataafterctrdata")
```

Which I found a little less clear.

Would you still prefer this approach?

Reply:
Yeah, I think I still prefer that, even if it's a bit stupid :-)

Reply:
Updated